t04_refactored.py

```diff
@@ -123,8 +123,57 @@ def team_1_adv():
 ###################################################################################
 
 def team_2_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/18CrlBERsLATii4ayTQ2jZwbzXCzwIUNAlY5zXM6osvI/edit?usp=sharing
+    Chase Phillips
+    Sonam Tsering
+    :return: none
+    """
+
+    print("It's Wednesday morning. You wake up at 10:30am and see the sun through your window")
+    print("Suddenly, you realize that you are almost late for your CSC 226 class at 10:40am ")
+    print("You know you need your morning coffee to get through the day, but you are almost late for class, "
+          "you could also just sleep in")
+    sleep(delay)
+    decision = input("Choose coffee, sleep, or class ")
+
+    if decision == "coffee" or decision == "Coffee":
+        # the good choice
+        print("You drink your coffee and run to class...")
+        sleep(delay)
+
+        isAlive = True
+    elif decision == "class" or decision == "Class":
+        # the bad choice
+        print("You manage to get in class at 10:41am")
+        sleep(delay)
+        isAlive = False
+    elif decision == "sleep" or decision == "Sleep":
+        # the neutral choice
+        print("You decide to sleep in, you miss class and you live")
+        quit()
+    else:
+        print("You can't do that, try again!")
+        sleep(3)
+        print()
+        team_2_adv()
```

Comment:
This restarts the function. Why'd we do this?

All the functions get called in main already, so you don't need to call it, really, ever?